test/helpers.ts

```diff
@@ -271,6 +271,13 @@ export function assertHasUpdate(
   return found!;
 }
 
+export function getUpdate(updates: Update[], path: string): Update {
```

Comment:
`assertHasUpdate` returns the found update as well. All the places you are using `getUpdate`, you are using `assertHasUpdate` immediately before.